django_celery_monitor/__init__.py

```diff
@@ -10,7 +10,7 @@
 
 from collections import namedtuple
 
-__version__ = '1.1.2+dj32fixes'
+__version__ = '1.1.3+dj32fixes'
```

Comment:
I might add a suffix to the end rather than switch to 1.1.3. 1.1.2 was the last official django-celery-mon, then we were adding on +whatever to indicate the deltas from that release...

Reply:
:+1: